snapshots/windows/utilities.go

```diff
@@ -0,0 +1,16 @@
+// +build windows
+
+package windows
+
+import (
+	"context"
+
+	"github.com/containerd/containerd/log"
+	"github.com/containerd/containerd/snapshots/storage"
+)
+
+func rollbackWithLogging(ctx context.Context, t storage.Transactor) {
```

Comment:
nit: this is only used in `snashots/windows/widnows.go` it could just be in that file.

Reply:
removed it, as logging the rollback failures is actually not very useful at all (it only fails if the transaction is already committed, at which point we don't care)